File: Lista3/Exec4/src/exec4/DVD.java
/**
 * Classe DVD
 */
public class DVD extends Estoque {

    /**
     * expande a classe Estoque com as info do DVD
     * @param nome
     * @param num
     * @param qte
     */
    public DVD(String nome, double num, int qte){
        super(nome, num, qte);
    }

    @Override
    public String toString(){
        String s = "Tipo: DVD\n";
        s += "Nome: " + getNome() + "\n";
        s += "Código de Barras: " + getNum() + "\n";
        s += "Quantia em Estoque: " + getQte() + "\n";
        return s;
    }

}

File: Lista4/exec4/DVD.py
from estoque import *

class DVD(estoque):

    #  * expande a classe Estoque com as info do DVD
    #  * @param nome
    #  * @param num
    #  * @param qte
    def __init__(self, nome: str, num: float, qte: int) -> None:
        super().__init__(nome, num, qte)

    def __str__(self) -> str:
        s = 'Tipo: DVD'
        s += "Nome: " + str(self.getNome()) + "\n"
        s += "Código de Barras: " + str(self.getNum()) + "\n"
        s += "Quantia em Estoque: " + str(self.getQte()) + "\n"
        return s
